src/background/stores/card-selection-store.ts

```diff
@@ -74,11 +80,16 @@ export class CardSelectionStore extends BaseStoreImpl<CardSelectionStoreData> {
 
         rule.cards[payload.resultInstanceUid] = !rule.cards[payload.resultInstanceUid];
 
+        // whenever a card is selected, the visual helper is enabled
```

Comment:
We strongly prefer to avoid using comments in our code and I'm not sure if it's necessary in this case.

I think we can create a variable that represents the "isSelected" property and instead the if check would read fairly well. This would also simplify some of your code above and make it more readable as you reach further into the objects more than once.